runbooks/patterns/delegation/delegate-failure.runbook.md

```
@@ -0,0 +1,32 @@
+    description: Delegated work fails, STOP with message
+    commands:
+      - rd run --prompted delegate-failure.runbook.md
+      - rd fail
+    result: STOP
+---
+
+# Delegation Failure
+
+Child failure propagates STOP to the parent.
+
+## 1. Delegate work
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)